src/qibolab/result.py

```diff
@@ -97,26 +68,26 @@ def to_dict_probability(self, state=1):
         elif state == 0:
             return {"probability": self.ground_state_probability}
 
-    def compute_average(self):
+    @property
+    def average(self):
         """Perform average over i and q"""
-        return AveragedResults(self.array.i.mean(), self.array.q.mean())
+        return AveragedResults.from_components(self.i.mean(), self.q.mean())
 
-    def to_dict(self, average=True):
-        """Serialize output in dict.
-        Args:
-            average (bool): If `True` returns a dictionary of the form
-                            {'MSR[V]' : v, 'i[V]' : i, 'q[V]' : q, 'phase[rad]' : phase}.
-                            Where each value is averaged over the number shots. If `False`
-                            all the values for each shot are saved.
-        """
-        results = self.compute_average() if average else self
+    def to_dict(self):
```

Comment:
Sometimes I called something like this `.raw`, rather than `.dict`, but the best name is up to you. When they are not much different, I'm terrible with this kind of choices...

Reply:
I understand that this should be a property, however `.dict` sounds a bit strange, given that it is already a keyword in python. What about `serial`?

Reply:
Indeed, as I said above, for the same reason I used `.raw`, but there is no very meaningful reason for which is much better than any other name. Just choose your favorite ^^

Reply:
Lets go for `serial` then :)